docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -85,157 +88,271 @@ See [examples](#content-examples) of OSCAL catalogs.
+
+A *parameter* in OSCAL (tagged as `<param>` in XML or, in JSON or YAML, as an object member of a `parameters` array) corresponds to and supports the construct called *controlled parameter value* (CPV) in NIST SP 800-53A, while not being limited exclusively to that use.
+
+*Note*: In this data, they appear with the controls in which they are used, but in OSCAL data, parameters may also be defined elsewhere, including at the group level or across a catalog, thus facilitating their consolidation and reuse.
+
+*Note also*: for succinctness, the code sampled here shows only the first four of the parameters defined for AC-1 in the document (version) it was derived from.
+
+{{<highlight xml "nowrap=false" >}}
+<param id="ac-1_prm_1">
+    <prop name="aggregates" ns="http://csrc.nist.gov/ns/rmf"
+        value="ac-01_odp.01"/>
+    <prop name="aggregates" ns="http://csrc.nist.gov/ns/rmf"
+        value="ac-01_odp.02"/>
     <label>organization-defined personnel or roles</label>
-  </param>
-  <param id="ac-1_prm_2">
-    <label>organization-defined frequency</label>
-  </param>
-  <param id="ac-1_prm_3">
-    <label>organization-defined frequency</label>
-  </param>
-  <prop class="label">AC-1</prop>
-  <part class="statement" id="ac-1_smt">
-    <p>The organization:</p>
-    <part class="item" id="ac-1_smt.a">
-      <prop class="label">a.</prop>
-      <p>Develops, documents, and disseminates to <insert param-id="ac-1_prm_1"/>:</p>
-      <part class="item" id="ac-1_smt.a.1">
-        <prop class="label">1.</prop>
-        <p>An access control policy that addresses purpose, scope, roles,
-           responsibilities, management commitment, coordination among
-           organizational entities, and compliance; and</p>
+</param>
+<param id="ac-01_odp.01">
+    <prop name="label" class="sp800-53a" value="AC-01_ODP[01]"/>
+    <label>personnel or roles</label>
+    <guideline>
+        <p>personnel or roles to whom the access control policy is to be
+        disseminated is/are defined;</p>
+    </guideline>
+</param>
+<param id="ac-01_odp.02">
+    <prop name="label" class="sp800-53a" value="AC-01_ODP[02]"/>
+    <label>personnel or roles</label>
+    <guideline>
+        <p>personnel or roles to whom the access control procedures are
+        to be disseminated is/are defined;</p>
+    </guideline>
+</param>
+<param id="ac-01_odp.03">
+    <prop name="alt-identifier" value="ac-1_prm_2"/>
+    <prop name="label" class="sp800-53a" value="AC-01_ODP[03]"/>
+    <select how-many="one-or-more">
+        <choice>organization-level</choice>
+        <choice>mission/business process-level</choice>
+        <choice>system-level</choice>
+    </select>
+</param>
+[... params ac-01_odp.04 through ac-01_odp.08 ...]
```

Comment:
Should we include this if the sample is already truncated and disjoint in the new structure introduced from this PR?

```suggestion
```

Reply:
Discussed this when we met prior, this is now obsolete.